src/sass/_common/base/media-queries.scss

```diff
@@ -19,6 +19,9 @@
 }
 
 @media screen and (max-width: 768px) {
+    .center-text-m {
```

Comment:
this class is used throughout the code with the assumption that it will only be centered in mobile screen size, can you instead create a new class `center-text-p` so we don't break others?